hackclub.com.yaml

```
@@ -2459,6 +2459,10 @@ som-mail-system:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+souptik:
+   - ttl: 600
+     type: CNAME
+     value: souptik-samanta.github.io/.
```

```suggestion
     value: souptik-samanta.github.io.
```

DNS records can't have paths (`/`)

---

I will do it and let you know
One issue could you go to the link I have in value that link is working when going from browser but 
Previously you had approved and merger it but when I was going to the 
Subdomain.hackclub.com(The subdomain is what I have that time) it was showing not found but was redirecting to gihub pages where a 404 error code was showing what might be the issue ?
Please let me know and I will remove the /
@/muirrum

---

you need to configure the CNAME in github pages.